docs/data_schema.md

```diff
@@ -220,41 +217,63 @@ Contains all result data from the static analysis; see description below
 ### `results` object
 
 #### `files`
-List of static analysis results, one per file contained in the analyzed package tarball. Files are enumerated in lexical order. Symlinks or special files such as device files, sockets and pipes are excluded. Each item is corresponds to a SingleResult object in Go; see description below.
+List of static analysis results, one per file contained in the analyzed package tarball. Files are enumerated in lexical order. Symlinks or special files such as device files, sockets and pipes are excluded. Each item is corresponds to a FileResult object in Go; see description below.
```

Comment:
nit: `Each item is corresponds` -> `Each item corresponds`

Reply:
done, thanks